docs/general/TestNet/node_level_architecture.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 2
+---
+
+# Node-level architecture
+
+
+
+## Node Types
+
+!!! TODO - where or why is this used
+!!! TODO - link to documentation
+
+### Suzuka node
```

Comment:
Suzuka is the name of the network, it has various types nodes, some of which are listed below.

Reply:
Should we rename this to Bardock to keep in line with what is being communicated externally?

Reply:
OK this seems I misunderstood. Please adjust node names to reflect the setup. 

Isn't suzuka and bardock both instances of the same testnet setup?

Reply:
according to @mzabaluev the node names are now 
- full node
- validator node
- DA light node
- follower node
- archival node

the names suzuka node and bardock node seem abandoned, correct @mzabaluev ?